service/service.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+var (
+	EnableStrongReadConcern    = false
+	EnableMajorityWriteConcern = true
```

Comment:
I think this should be configurable. If for any reason we needed to change these values we would have to make a new release, if its config it will be far easier.

Reply:
made the changes